module/actuation/Servos/src/Servos.hpp

```diff
@@ -58,10 +61,21 @@ namespace module::actuation {
                         return;
                     }
 
-                    // Runs an emit for each servo
-                    NUClear::util::unpack((emit<Task>(std::make_unique<Elements>(
-                                               group.servos.at(utility::actuation::ServoMap<Elements>::value))),
-                                           0)...);
+                    (
+                        // Runs an emit for each servo, if that servo was included in the map
+                        [&] {
+                            // Check if the Task includes this servo
+                            if (group.servos.contains(utility::actuation::ServoMap<Elements>::value)) {
+                                // Emit a Task for the servo if the map contains a value for it
+                                emit<Task>(std::make_unique<Elements>(
+                                    group.servos.at(utility::actuation::ServoMap<Elements>::value)));
+                            }
+                            else {  // if a servo was not filled in the map for this group, log it
+                                log<NUClear::DEBUG>("Requested a servo group Task but did not provide values for servo",
+                                                    utility::actuation::ServoMap<Elements>::value);
```

Comment:
I'd drop this to TRACE, since it's probably going to show up a lot.
Also being that it's a smart enum is this printing the actual name rather than just a number?

Reply:
It's printing a number

Reply:
In that case, you can change it to make it print out a string instead using the smart enums